### ISSUE TITLE
Error when autoprofiling

### ISSUE THREAD

**Issue by jgbarah** [chaoss/grimoirelab-sortinghat#122]
I see this error:

```
$ sortinghat -u grimoirelab --host localhost -d grimoirelab_sh autoprofile chaoss:manual git github
Traceback (most recent call last):
  File "/tmp/gl/bin/sortinghat", line 6, in <module>
    exec(compile(open(__file__).read(), __file__, 'exec'))
  File "/home/jgb/src/jgbarah/sortinghat/bin/sortinghat", line 181, in <module>
    code = main()
  File "/home/jgb/src/jgbarah/sortinghat/bin/sortinghat", line 93, in main
    code = cmd.run(*args.cmd_args)
  File "/home/jgb/src/jgbarah/sortinghat/sortinghat/cmd/autoprofile.py", line 75, in run
    code = self.autocomplete(sources)
  File "/home/jgb/src/jgbarah/sortinghat/sortinghat/cmd/autoprofile.py", line 111, in autocomplete
    self.display('autoprofile.tmpl', identity=identity)
UnboundLocalError: local variable 'identity' referenced before assignment
```

Identities JSON file, exported as `sortinghat -u grimoirelab --host localhost -d grimoirelab_sh export --identities > /tmp/identities.json`: [identities.json.gz](https://github.com/chaoss/grimoirelab-sortinghat/files/1723526/identities.json.gz)

**Comment by jgbarah** [comment]
Some more info. It seems this happens only when some tag in the list for autoprofiles is not in the SH database. In this case, the tag in the database is `CHAOSS:manual`, not `chaoss:manual`. If I change that in the call to SortingHat, there is no error.

**Comment by jgbarah** [comment]
Looking at the code in `autoprofile.py` (function `autocomplete()`), the problem seems to be that only identities for the specified sources are collected. For some reason, if some of the uuids has no identity for those data sources (I assume that happens when the only identity is `chaoss:manual`, in this case), the code fails.

In the short term, this could be fixed by using the selected profile (which could be None) instead of identity in 

```
self.display('autoprofile.tmpl', identity=identity)
```

In the long term, I think this should be fixed by selecting *all* ids, not only those of the specified data sources, although the profile should still be filled in, with preference, from the ids in the data sources, if any.

**Comment by jgbarah** [comment]
See an implementation for the first fix proposed above in #127.

**Comment by sduenas** [comment]
Autoprofile is not supported anymore.
